components/search/HotelSearch/FilterButton.tsx

```diff
@@ -4,14 +4,14 @@ import React, { useState, useEffect } from "react";
 import { Filter } from "lucide-react";
 
 const FilterButton: React.FC = () => {
-  const [buttonOffset, setButtonOffset] = useState(90); 
-  const [isSticky, setIsSticky] = useState(false); 
-  const [isMobile, setIsMobile] = useState(false); 
+  const [buttonOffset, setButtonOffset] = useState(90);
+  const [isSticky, setIsSticky] = useState(false);
+  const [isMobile, setIsMobile] = useState(false);
 
   useEffect(() => {
     // Function to detect screen size
     const handleResize = () => {
-      setIsMobile(window.innerWidth < 768); 
+      setIsMobile(window.innerWidth < 1200);
     };
 
     handleResize();
```

Comment:
sm is limiting as different pages use different sizes, and in this case, it does not work for the limitation. And since 1200 is the same as the nav breaking point, and seeing as how the nav mobile perspective interferes with the filter button, we get this change.